apps/src/p5lab/AnimationTab/scrollable-list.module.scss

```diff
@@ -0,0 +1,8 @@
+.root {
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
+
+.margins {
+  margin: 4;
```

Comment:
Nit: 4px? Does this work without specifying units?

Reply:
It most certainly did not, but it should now!